docs/source/tutorials/transfer_learning/transfer_learning.rst

```diff
@@ -0,0 +1,62 @@
+Using Syne Tune for Transfer Learning
+======================================
+
+Transfer learning allows us to speed up our current optimisation by learning from related optimisation runs. For instance, imagine we want to change from a smaller to a larger model. We already have a collection of hyperparameter evaluations for the smaller model. Then we can use these to guide our hyperparameter optimisation of the larger model, for instance by starting with the configuration that performed best. 
+
+Syne Tune includes implementations of several transfer learning schedulers; a list of available schedulers is given  `here <../../getting_started.html#supported-hpo-methods>`_. In this tutorial we look at three of them: 
```

Comment:
Did you also get transfer BO working, where a multi-task GP is fit to all data? I recall you asked about this, and I did some fixes. It would be nice to have this here as well.

Reply:
It ran into issues when we wanted to modify the fidelities between tasks and giving constants in the configuration space. We ended up using the BoTorch version instead https://github.com/sighellan/syne-tune/blob/botorch-transfer/syne_tune/optimizer/schedulers/searchers/botorch/botorch_transfer_searcher.py